api/v1alpha1/nonadminbackup_types.go

```diff
@@ -74,6 +74,68 @@ type VeleroDeleteBackupRequest struct {
 	Namespace string `json:"namespace,omitempty"`
 }
 
+// TODO
+type DataMoverVolumeBackups struct {
+	// number of DataUploads related to this NonAdminBackup's Backup
+	// +optional
+	Total int `json:"total,omitempty"`
```

Comment:
```// total number of DataUploads related to this NonAdminBackup's Backup across all phases.```

We have here NonAdminBackup and Backup wording, so dropping this wording from other descriptions to make them more informative?